### PR TITLE
[2.13.x] DDF-3941 - Moved intrigue from using the SessionManagementService Endpoint under /services

### PR DESCRIPTION
**Pull request by blen-desta**
Backport of https://github.com/codice/ddf/pull/3382
#### What does this PR do?
- Extracted Session management implementation into a service
- Changed current Session management to use extracted service
- Added new endpoint under /search/catalog/internal
- Changed code under  to use new endpoint

#### Who is reviewing it? 
@SmithJosh @bakejeyner 

#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@brendan-hofmann
@coyotesqrl

#### How should this be tested? (List steps with links to updated documentation)
Make sure that session renewal, expiry and invalidation works from both intrigue and the admin console.

#### What are the relevant tickets?
[DDF-3941](https://codice.atlassian.net/browse/DDF-3941)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by blen-desta** [comment]
build now

**Comment by SmithJosh** [comment]
Manually verified that session renewal, expiry and invalidation work from both Intrigue and the Admin UI, and the countdown does not go negative.

**Comment by blen-desta** [comment]
build now

**Comment by blen-desta** [comment]
build now

File: platform/security/session-management-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<blueprint xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.2.0"
+           xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+    <ext:property-placeholder/>
```
Comment (bakejeyner) [inline platform/security/session-management-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: ❓ What does this do?

File: platform/security/session-management-impl/pom.xml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Embed-Dependency>
+                            ddf-security-common,
+                            platform-util,
+                            platform-util-unavailableurls
+                        </Embed-Dependency>
```
Comment (bakejeyner) [inline platform/security/session-management-impl/pom.xml]: ❓ Is there a reason are we embedding, and not wrapping, `ddf-security-common` and `platform-util`. I would think those are being used almost everywhere and would be wrapped.

**Pull request merged**